src/components/TransactionDetails/TransactionDetailsDrawer.tsx

```
@@ -102,220 +145,257 @@
+                        transaction.cancelledDate && (
+                            <>
+                                {transaction.cancelledDate && (
+                                    <PaymentInfoRow
+                                        label="Cancelled"
+                                        value={formatDate(transaction.cancelledDate as Date)}
+                                        hideBottomBorder={
+                                            !transaction.fee && !transaction.memo && !transaction.attachmentUrl
+                                        }
+                                    />
+                                )}
+                            </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)